README.md

```diff
@@ -82,7 +82,19 @@ If you don't want to use foreman, you have to run the rails server (Puma) and th
 
 If adding or modifying tests that will make new calls to the Google Maps APIs, you'll need to run with a valid `GOOGLE_API_KEY`, both to get your tests to pass, and also to refill the cached test data for use in later runs (including CI). See the discussion in `spec/rails_helpers.rb` for instructions on how to run with a live API Key.
 
-## Running the app locally
+## Running cron or a scheduled Lambda to promote rides
```

Comment:
Do you think _most_ app developers need to know this? If not, perhaps it could go below "Using the app locally" so as not to overwhelm.

Reply:
Yeah I wondered that, and to some degree these were notes to myself. Thing is, if you run the app locally, and don't do this, you also may have no idea why scheduled rides aren't being promoted to 'waiting assignment,' which makes them available to drivers and is the main purpose of the cron/lambda. If you don't mind I'm going to leave it in--along with the modal, rewriting the README--and perhaps moving stuff like this into a reworked wiki and just references it--is going to be a priority post-election, when I'm hoping to go into a slow-and-steady maintenance routine--hopefully with anyone else who wants to push toward 2020 :-)